ecgtools/core.py

```diff
@@ -376,15 +384,15 @@ def batch(seq):
             result_batch = dask.delayed(batch)(filepaths[i : i + nbatches])
             results.append(result_batch)
     else:
-        results = [_parse_file_attributes(filepath, parser) for filepath in filepaths]
+        results = [_parse_file_attributes(filepath, parser, local_attrs) for filepath in filepaths]
 
     if dask.is_dask_collection(results[0]):
         results = dask.compute(*results)
         results = list(itertools.chain(*results))
     return pd.DataFrame(results)
 
 
-def _parse_file_attributes(filepath: str, parser: callable = None):
+def _parse_file_attributes(filepath: str, parser: callable = None, local_attrs: dict = {}):
```

Comment:
It's my understanding that since we are using the `partial` function [here](https://github.com/NCAR/ecgtools/pull/5/files#diff-51e88db5c0bcb5cf99f4aa7dc3f9c7d6R297), we don't need to modify the signature of `_parse_file_attributes`.

Reply:
Thank you, I had forgotten to remove these references to local_attrs when we made that switch.  It should be fixed in the next push.